src/python/strelka/scanners/scan_html.py

```diff
@@ -12,6 +12,7 @@ class ScanHtml(strelka.Scanner):
     """
     def scan(self, data, file, options, expire_at):
         parser = options.get('parser', 'html.parser')
+        link_limit = options.get('max_links', 50)
```

Comment:
The pattern in the other scanners is to have the config option match the internal variable.

Reply:
Yeah - good catch. Modified options name.